docset/winserver2019-ps/defender/Get-MpPerformanceReport.md

```diff
@@ -0,0 +1,265 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: MSFT_MpPerformanceReport.cdxml-help.xml
+Module Name: Defender
+ms.date: 08/25/2021
+online version: https://docs.microsoft.com/powershell/module/defender/get-mpperformancereport?view=windowsserver2019-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Get-MpPerformanceReport
+---
+
+# Get-MpPerformanceReport
+
+## SYNOPSIS
+Analyzes and reports on MDAV performance recording
+
+## SYNTAX
+
+```
+Get-MpPerformanceReport    [-Path] <String>
+[-TopScans <Int32>]
+[-TopFiles  <Int32>
+[ -TopScansPerFile <Int32>]
+[-TopProcessesPerFile  <Int32>  
+	[-TopScansPerProcessPerFile <Int32>]
+	]
+] 
+[-TopExtensions  <Int32>
+[-TopScansPerExtension <Int32>
+[-TopProcessesPerExtension <Int32>
+	[-TopScansPerProcessPerExtension <Int32>]
+	]
+
+[-TopFilesPerExtension  <Int32>
+	[-TopScansPerFilePerExtension <Int32>]
+	]
+] 
+[-TopProcesses  <Int32>
+	[-TopScansPerProcess <Int32>]
+	[-TopExtensionsPerProcess <Int32>
+		[-TopScansPerExtensionPerProcess <Int32>]
+		]
+[-TopFilesPerProcess  <Int32>
+	[-TopScansPerFilePerProcess <Int32>]
+]
+```
+
+## DESCRIPTION
+
+The `Get-MpPerformanceReport` cmdlet analyzes a previously collected Microsoft Defender Antivirus performance recording ([New-MpPerformanceRecording](New-MpPerformanceRecording.md#new-mpperformancerecording)) and reports the file paths, file extensions, and processes that cause the highest impact to Microsoft Defender Antivirus scans.
+
+The performance analyzer provides an insight into problematic files that could cause a degradation in the performance of Microsoft Defender Antivirus. This tool is provided "AS IS" and is not intended to provide suggestions on exclusions. Exclusions can reduce the level of protection on your endpoints. Exclusions, if any, should be defined with caution.
+
+For more information on the Performance analyzer, see Performance Analyzer docs.
```

Comment:
```suggestion
For more information on the performance analyzer, see the  [Performance Analyzer](/windows-hardware/test/wpt/windows-performance-analyzer) topic.
```
```

Reply:
partially fixed; to include the relative link (in other instances also)